db/migrate/20160801193239_add_identity_uuid.rb

```diff
@@ -0,0 +1,6 @@
+class AddIdentityUuid < ActiveRecord::Migration
+  def change
+    add_column :identities, :uuid, :string, null: false
```

Comment:
I would rather not. Calling it the `user_uuid` suggests to me that the uuid is for the user. It's not. It's for the user's identity.